circle.yml

```diff
@@ -0,0 +1,15 @@
+# circle.yml
+machine:
+  services:
+    - docker
+  environment:
+    EXPOSED_PORT: 8080
+
+dependencies:
+  override:
+    - pip install docker-compose
+    - EXPOSED_PORT=8080 docker-compose build
+
+test:
+  override:
+    - docker-compose run --rm web pytest
```

Comment:
Just FYI: There's no tests yet. #1 calls out what @mariusvniekerk and I were thinking as a smoke test.